src/evaluation_system/model/solr.py

```diff
@@ -251,6 +251,8 @@ def _facets(self, latest_version=False, facets=None, **partial_dict):
                     "uri",
                     "file",
                     "file_name",
+                    "time_aggregation",
+                    "grid_label",
```

Comment:
Are you sure about grid_label to be ignored? This is a facet in gems and Chris wanted me to include it.

Reply:
Ok I'll take in out.